src/scib_metrics/_graph_connectivity.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_matrix
+from scipy.sparse.csgraph import connected_components
+
+
+def graph_connectivity(X: csr_matrix, labels: np.ndarray) -> float:
+    """Quantify the connectivity of the subgraph per cell type label.
+
+    Parameters
+    ----------
+    X
+        Array of shape (n_cells, n_cells) with non-zero values
```

Comment:
do you want to assert the k nearest neighbors?

Reply:
left as a todo as we have to make global utils for this